AdobeStockImage/Model/SaveImage.php

```diff
@@ -88,6 +95,7 @@ public function __construct(
         DocumentToAsset $documentToAsset,
         SaveAssetKeywordsInterface $saveAssetKeywords,
         DocumentToKeywords $documentToKeywords,
+        SetLicensedInMediaGalleryGrid $setLicensedInMediaGalleryGrid,
```

Comment:
This dependency will be removed in https://github.com/magento/adobe-stock-integration/issues/1146